File: INICIANTE/1013/main.dart
import 'dart:io';

int maior(int a, int b) {
  return (a + b + (a - b).abs()) ~/ 2;
}

void main() {
  var [a, b, c] = stdin.readLineSync()!.split(' ').map(int.parse).toList();
  print('${maior(maior(a, b), c)} eh o maior');
}

File: INICIANTE/1013/main.cpp
#include <iostream>
using namespace std;

int maior(int a, int b) {
    return (a + b + abs(a - b)) / 2;
}

int main() {
    int a, b, c;
    cin >> a >> b >> c;
    cout << maior(maior(a, b), c) << " eh o maior" << endl;
    return 0;
}
